terraform-beyond-layer0/example-3/env-dev/main.tf

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
One of the main take-aways from example 3 that I wanted to get across is how to structure your configuration files, such that you have environment specific state files.